vpr/test/test_map_lookahead_serdes.cpp

```diff
@@ -1,62 +0,0 @@
-#include "catch.hpp"
```

Comment:
Why did this test get deleted?

Reply:
This was based on the previous router_lookahead_map code, I would need to readapt the test with the code merged from the connection box lookahead map and the upstream lookahead map. It still is in the TODO list.